pkg/compactor/bucket_compactor.go

```diff
@@ -280,13 +280,13 @@ func (c *BucketCompactor) runCompactionJob(ctx context.Context, job *Job) (shoul
 
 	defer func() {
 		elapsed := time.Since(jobBeginTime)
-		level.Info(jobLogger).Log("msg", "compaction job finished", "success", rerr == nil, "duration", elapsed, "duration_ms", elapsed.Milliseconds())
 
-		// Leave the compact directory for inspection if it is a halt error
-		// or if it is not then so that possibly we would not have to download everything again.
-		if rerr != nil {
-			return
+		if rerr == nil {
+			level.Info(jobLogger).Log("msg", "compaction job succeeded", "duration", elapsed, "duration_ms", elapsed.Milliseconds())
```

Comment:
Why is it necessary to add the duration once in seconds and once in milliseconds?
If we need milliseconds precision, wouldn't `duration_ms` alone be sufficient?

Reply:
It was inherited from Thanos. I agree we can drop one of the two.